examples/schedule/outputs.tf

```
@@ -0,0 +1,14 @@
+output "team" {
+  description = "Opsgenie Team"
+  value       = module.team
```

These may need to be changed to `module.owner_team`

```suggestion
  value       = module.owner_team
```